amplify/backend/function/risestandstrongcb34053a/src/routes/shifts.js

```diff
@@ -201,11 +201,29 @@ router.get('/:startTimestamp', async (req, res) => {
  *      responses:
  *         "200":
  *            description: Success
- *         "404":
+ *         "400":
  *            description: startTimestamp not found
  */
 router.put('/:startTimestamp', async (req, res) => {
-   res.end();
+   const startTimestamp = req.params.startTimestamp;
+   const parsedStartTS = parseInt(startTimestamp);
+   if(Number.isNaN(parsedStartTS)) {
+      res.status(400).send({
+         error: 'Invalid query parameters, startTimestamp is required and must be a Number.',
+      });
+   } else {
+      const shiftBody = {
+         startTimestamp: parsedStartTS,
+         ...req.body
+      }
+      try {
+        await putShift(shiftBody);
+        res.send(200);
```

Comment:
Change this to res.end(). I got an error that res.send() is deprecated and res.end() sends the response with a 200.
```suggestion
        res.end();
```